subprojects/gradle-plugin/src/main/java/org/spongepowered/gradle/vanilla/internal/model/GroupArtifactVersion.java

```diff
@@ -30,16 +30,26 @@
 @Value.Immutable(builder = false)
 public abstract class GroupArtifactVersion {
 
-    public static GroupArtifactVersion of(final String group, final String artifact, final @Nullable String version) {
-        return new GroupArtifactVersionImpl(group, artifact, version);
+    public static GroupArtifactVersion of(
```

Comment:
Don't replace the existing method, add a new overload with the classifier parameter.